yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -162,8 +162,10 @@ public class Document private constructor(
         // TODO: also apply to root
     }
 
+    // NOTE(7hong13): original comment from JS-SDK:
```

Comment:
@hackerwins it's not directly related to this PR... but I changed the comment here. 
(purges -> delete)